packages/worker/src/api/routes/global/tests/email.spec.ts

```diff
@@ -6,7 +6,7 @@ import { EmailTemplatePurpose } from "../../../../constants"
 describe("/api/global/email", () => {
   const config = new TestConfiguration()
 
-  beforeAll(async () => {
+  beforeEach(async () => {
     await config.beforeAll()
```

Comment:
Is this intentional , `beforeEach` calling `beforeAll`? The beforeAll is generally a test level setup, that does any heavier lifting.

Reply:
It does not look intentional here, as there is only one test in the file 😅 In some other files it was required, as the `.beforeAll` is creating a new tenant, and there were conflicts with tests that required a clean instance

Reply:
I think we should try to be more clear on the intent in this case, as it can set a misleading pattern to new devs that we must use the `beforeAll` helpers in `beforeEach` blocks. 

An alternative might be to add a helper to config, `reset`, and we can use as
```ts
beforeEach(async () => {
   await config.reset()
})
```

This can under the hood just call `beforeAll`, it's just a bit more clear on the reason why.